.github/CODEOWNERS

```diff
@@ -316,4 +316,5 @@
 /templates/python/* @adashen @blackchoey @frankqianms 
 /templates/python/custom-copilot-basic @frankqianms @blackchoey
 /templates/python/custom-copilot-rag-azure-ai-search @frankqianms @blackchoey
+/templates/python/custom-copilot-rag-customize @frankqianms @blackchoey
```

Comment:
Since we already have line 316, maybe we don't need to add this line any more?

Reply:
ok, will remove it